Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -33,6 +33,11 @@ private float MaxFrameTimeInSeconds
             get { return (MaxFrameTime / 1000); }
         }
 
+        /// <summary>
+        ///  Whether to create mesh colliders. If unchecked, mesh colliders will be empty and disabled.
+        /// </summary>
+        public bool CreateMeshColliders = true;
```

Comment:
That's a good point, and it seems like more trouble than its worth. I'm fine with things as they are right now.